src/Symfony/Component/Clock/MockClock.php

```diff
@@ -47,6 +47,11 @@ public function sleep(float|int $seconds): void
         $this->now = (new \DateTimeImmutable($now, $timezone))->setTimezone($timezone);
     }
 
+    public function setTo(\DateTimeImmutable $now): void
```

Comment:
I like the proposal

What about naming it "to()" ?

Reply:
Sure. I will change it later today

Reply:
I'm not sure if that is more clear, it seems to miss the "verb".

If you don't like `setTo()`, what about `changeTo()` or `setNow()` maybe?

Reply:
`setNow()` sounds good to me as we already have a `now()` that returns the current time.

Reply:
Yes, from all these options `setNow()` sounds best to me. I'll wait a bit for more feedback and finish it

Reply:
"setNow" can be understood as _"set the clock time to now"_, so it could be confusing.

Reply:
I like `setNow` 👍 

Thanks for the discussion!